src/content/docs/en/guides/integrations-guide/vue.mdx

```diff
@@ -211,3 +211,21 @@ export default defineConfig({
   integrations: [vue({ devtools: true })],
 });
 ```
+
```

Comment:
It will proably require the Since component there but wait for others' confirmation